LondonBritishLibrary/orient/BLorient12860.xml

```diff
@@ -0,0 +1,207 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BLorient12860" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>The life of Yāred, Malkǝʾ of Yāred</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <collection>Oriental</collection>
+                        <idno>BL Oriental 12860</idno>
+                        <altIdentifier><idno>Strelcyn 55</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1ra" to="26vb"/> 
+                            <title ref="LIT1512GadlaY"/>
+                            <note>With many variations to <bibl><ptr target="bm:ContiRossini1904YaredPantalewonT"/></bibl>.</note>
+                            <msItem xml:id="ms_i1.1">
+                                <locus from="1ra"/>
+                                <title ref="LIT1303Dersan"/>
+                                <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ድርሳን፡ ዘቅዱስ፡ ያሬድ፡ ቀርነ፡ ቤተ፡ ክርስቲያን፡ ዘይጼውዖሙ፡ ለሃይማናን፡ ከመ፡
```

Comment:
I suppose that መሃይምናን፡ must be meant, so maybe it would be good to have a "sic" here. The question is if the "error" is found in the manuscript or only in Strelcyn's catalogue (I checked and you have copied his text faithfully)

Reply:
I added a `<sic resp="CH">`, even though I could not check it by myself, because images are still not available.